packages/material-ui/src/ExpansionPanel/ExpansionPanel.js

```diff
@@ -197,7 +201,7 @@ ExpansionPanel.propTypes = {
    */
   onChange: PropTypes.func,
   /**
-   * @ignore
+   * If `true`, rounded corners are disabled.
```

Comment:
We could keep ignoring this but I would argue that it's better to document this here since the classes reference it directly ("rounded is applied if `square={false}`").